src/Common/Misc/ConvertProxyProperty/index.js

```diff
@@ -9,7 +9,7 @@ const typeMapping = {
   ProxyEditorPropertyWidget: 'ProxyEditorPropertyWidget',
 };
 
-function isGroupWidget(widgetType) {
+export function isGroupWidget(widgetType) {
```

Comment:
Since it was like that, let's keep it. But I believe we should not do that unless the value is already a string.